test/data/testinit.js

```diff
@@ -48,6 +49,32 @@ QUnit.assert.t = function( a, b, c ) {
 	}
 
 	this.deepEqual( f, q.apply( q, c ), a + " (" + b + ")" );
+}
+
+/**
+ * Asserts that a select matches the given IDs.
+ * The select is not bound by a context.
+ * @param {String} a - Assertion name
+ * @param {String} b - Sizzle selector
+ * @param {String} c - Array of ids to construct what is expected
+ * @example t("Check for something", "//[a]", ["foo", "bar"]);
+ * @result returns true if "//[a]" return two elements with the IDs 'foo' and 'bar'
+ */
+QUnit.assert.t = function( a, b, c ) {
+	match( a, b, c, undefined );
+};
+
+/**
+ * Asserts that a select matches the given IDs.
+ * The select is performed with the `#qunit-fixture` context.
+ * @param {String} a - Assertion name
+ * @param {String} b - Sizzle selector
+ * @param {String} c - Array of ids to construct what is expected
+ * @example t("Check for something", "//[a]", ["foo", "bar"]);
+ * @result returns true if "//[a]" return two elements with the IDs 'foo' and 'bar'
+ */
+QUnit.assert.u = function( a, b, c ) {
```

Comment:
@markelog now I also see that I missed adjusting function names at lines [39][1] and [73][2] in the jsdoc. Once you decide if you want new names or not, I will fix that.

[1]: https://github.com/jquery/jquery/pull/2973/files#diff-289e07cdd24b4895b9f305d02f7ad25bR39
[2]: https://github.com/jquery/jquery/pull/2973/files#diff-289e07cdd24b4895b9f305d02f7ad25bR39